src/coreclr/src/inc/crosscomp.h

```diff
@@ -198,14 +196,14 @@ typedef struct _T_DISPATCHER_CONTEXT {
     PUCHAR NonVolatileRegisters;
 } T_DISPATCHER_CONTEXT, *PT_DISPATCHER_CONTEXT;
 
-#if defined(TARGET_UNIX) || defined(HOST_X86)
-#define T_RUNTIME_FUNCTION RUNTIME_FUNCTION
-#define PT_RUNTIME_FUNCTION PRUNTIME_FUNCTION
-#else
+#if defined(HOST_WINDOWS) && defined(HOST_X86)
```

Comment:
Here it just felt like it would be quicker to understand the code if the controlling logic matched.  So I reverse the order of the blocks to get it to match.

Reply:
If I understand what you did `!(defined(TARGET_UNIX) || defined(HOST_X86))` --> `defined(TARGET_WINDOWS) && !defined(HOST_X86)`

Reply:
Yeah it looks broken.  

It is odd to be written like it is.  The structure is defined twice in the file.  Once as `RUNTIME_FUNCTION` and once as `T_RUNTIME_FUNCTION`.